types/fbt/index.d.ts

```diff
@@ -222,7 +217,19 @@ interface FbtHookRegistrations {
 
 export function init(options: { hooks?: FbtHookRegistrations; translations: TranslationDict }): void;
 
-export const fbt: Fbt;
+interface FbtOutput {
+    fbt: string[];
```

Comment:
I am not sure why this was `string[]` before, should not it be `string` or maybe `ReactNode/ReactElement` instead ?

Reply:
```
<fbt desc="param example">
  Hello,
  <fbt:param name="name">{person.getName()}</fbt:param>.
</fbt>
// ^^^ the result will be an array
```

Reply:
but it's using it as a JSX element, not function call right ? so at least for `fb()`, should we change it to return string instead ? right now, a lot warnings show up when try to assign `fbt()` just to strings

Reply:
I did a couple tests, and this how the fbt result represented  in JS runtime:

```tsx
import { isValidElement } from 'react';
import {GenderConst, IntlVariations, fbs, fbt, init} from 'fbt';

const withParams = fbt('Hello, ' + fbt.param('name', 'retyui'), 'param example');

typeof withParams; // object
withParams.toString() // "Hello, retyui"
JSON.stringify(withParams) // "\"Hello, retyui\""
isValidElement(withParams) // true
```

@taozhou-glean What do you think will be the best return type for `fbt()`  ?

Reply:
I think `string` would be very convenient given that's probably the expected type in most cases. Tho with `fbs`, I think `ReactNode` might be more accurate, and it's also recommended to use `fbs` for simple strings and `fbt` when html included.

Reply:
ok, let's replace `string[]` with `string`